src/Tizen.Multimedia.Vision/MediaVision/InferenceModelConfiguration.cs

```diff
@@ -57,6 +58,7 @@ public class InferenceModelConfiguration : EngineConfiguration
         // The following strings are fixed in native and will not be changed.
         private const string _backendTypeOpenCV = "opencv";
         private const string _backendTypeTFLite = "tflite";
+        private const string _backendTypeArmNN = "armnn";
```

Comment:
Is default backend still opencv? If not, please change the description of `Backend` property.

Reply:
Yes, opencv is still default backend.